File: Python/Veri_yapilari/Stack.py
class Stack:
	def __init__(self):
		self.items = []
	def isEmpty(self):
		return self.items == []
	def push(self,item):
		return self.items.append(item)
	def peek(self):
		return self.items[len(self.items) - 1]
	def pop(self):
		return self.items.pop()

File: Ruby/Stack.rb
class Stack
	def initialize
		@liste = []
	end
	def length 
		@liste.length
	end
	
	def push(item) 
		@liste.push item
		self
	end
	def isEmpty
		if @liste.length == 0
			return true
		else
			return false
		end
	end
	def peek 
		 @liste[-1]
	end
	def pop
         @liste.pop
	end
	def show
		print "Liste :", @liste
	end
end
